tests/integration.rs

```diff
@@ -5921,6 +5921,132 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
+        fn setup_spdk_nvme() {
+            cleanup_spdk_nvme();
+
+            assert!(
+                exec_host_command_status("mkdir -p /tmp/test-vfio-user/nvme-vfio-user").success()
+            );
+            assert!(
+                exec_host_command_status("truncate /tmp/test-vfio-user/test-disk.raw -s 128M")
+                    .success()
+            );
+            assert!(
+                exec_host_command_status("mkfs.ext4 /tmp/test-vfio-user/test-disk.raw").success()
+            );
+
+            // Start the SPKD nvmf_tgt daemon to present NVMe device as a VFIO user device
+            Command::new("/usr/local/bin/spdk-nvme/nvmf_tgt")
+                .args(&["-i", "0", "-e", "0xFFFF", "-m", "0x1"])
```

Comment:
Do we want to trace SPDK with "-e" "0xFFFF"

Reply:
This -e is used for tracing purpose. I think that you do not need that for test.  You can just delete -e 0xFFFF.

Meaning:  
 -e, --tpoint-group-mask <mask>
                           tracepoint group mask for spdk trace buffers (default 0x0, bdev 0x8, nvmf_tcp 0x20, blobfs 0x80, thread 0x400, nvme_pcie 0x800, all 0xffff)

I do not think that you need to use this for testing purpose.